_includes/content/private-registry-image-path.md

```diff
@@ -0,0 +1,96 @@
+   {%-    assign component_name = page.imageNames[component_name] -%}
+   {%     if component[1].registry %}{% assign registry = component[1].registry | append: "/" %}{% else %}{% assign registry = page.registry -%} {% endif -%}
+   docker pull {{ registry }}{{ component_name }}:{{component[1].version}}
+   {%   endunless %}{% endif -%}
+   {% endfor -%}
+   ```
+
+1. Retag the images with the name of your registry `$REGISTRY` and `$IMAGE_PATH`.
+
+   ```bash
+   docker tag {{ operator.registry }}/{{ operator.image }}:{{ operator.version }} $REGISTRY/$IMAGE_PATH/{{ operator.image | split: "/" | last }}:{{ operator.version }}
+   {% for component in site.data.versions.first.components -%}
+   {%-  capture component_name %}{{ component[0] }}{% endcapture -%}
+
+   {%-  if page.imageNames[component_name] %}{% unless component_name contains "flannel" -%}
+   {%-    assign component_name = page.imageNames[component_name] -%}
+   {%     if component[1].registry %}{% assign registry = component[1].registry | append: "/" %}{% else %}{% assign registry = page.registry -%} {% endif -%}
+   docker tag {{ registry }}{{ component_name }}:{{component[1].version}} $REGISTRY/$IMAGE_PATH/{{ component_name | split: "/" | last }}:{{component[1].version}}
+   {%   endunless %}{% endif -%}
+   {% endfor -%}
+   ```
+
+1. Push the images to your registry.
+
+   ```bash
+   docker push $REGISTRY/$IMAGE_PATH/{{ operator.image | split: "/" | last }}:{{ operator.version }}
+   {% for component in site.data.versions.first.components -%}
+   {%-  capture component_name %}{{ component[0] }}{% endcapture -%}
+
+   {%-  if page.imageNames[component_name] %}{% unless component_name contains "flannel" -%}
+   {%-    assign component_name = page.imageNames[component_name] -%}
+   docker push $REGISTRY/$IMAGE_PATH/{{ component_name | split: "/" | last}}:{{component[1].version}}
+   {%   endunless %}{% endif -%}
+   {% endfor -%}
+   ```
+
+#### Run the operator using images from your registry image path
+
+Before applying `tigera-operator.yaml`, modify registry references to use your custom registry:
+
+```bash
+{% if page.registry != "quay.io/" -%}
+sed -ie "s?{{ page.registry }}/.*/?$REGISTRY/$IMAGE_PATH/?" tigera-operator.yaml
+{% endif -%}
+sed -ie "s?quay.io/.*/?$REGISTRY/$IMAGE_PATH/?" tigera-operator.yaml
+```
+{% comment %} The second 'sed' should be removed once operator launches Prometheus & Alertmanager {% endcomment %}
+
+If using a registry that requires user authentication, ensure that an image pull secret is configured for your custom registry. Set the enviroment variable `REGISTRY_PULL_SECRET` to the secret name.
+
+Then, add the image pull secret to the operator deployment spec:
+
+```bash
+sed -ie "/serviceAccountName: tigera-operator/a \      imagePullSecrets:\n\      - name: $REGISTRY_PULL_SECRET"  tigera-operator.yaml
+```
+
+For <b>Openshift</b>, after downloading all manifests modify the following to use your custom registry:
```

Comment:
Should we do tabs for this? Seems like that would be nice

Reply:
oh hah, this is already in a tab, so that would be crazy!

Reply:
Not really that crazy...it would be easy to miss OCP...so I did put in the inline tabs.